ontology-config/application-doid.properties

```diff
@@ -0,0 +1,33 @@
+# The ontology URI
```

Comment:
I'm ok with not including these comments everywhere, but the readme should probably point to this specific properties file as the example. The others (without comments) would be confusing to encounter first.